### DISCUSSION TITLE
Plus code search speed

### DISCUSSION THREAD

**Discussion by DolomitesTom** [devemux86/cruiser#286] [Q&A] [answered]
The plus code search takes very long. Could this be improved by performance?

**Comment by devemux86** [comment]
Do you mean global or local plus codes?
- Global plus codes (11 characters): "7MJCH93V+7F" (completely offline)
- Local plus codes (7 characters + location): "H93V+7F Kolkata" (require online search)

It currently also depends on the enabled providers in search settings.
We can probably do it faster when the parser recognizes a plus code.

**Comment by devemux86** [comment]
>  We can probably do it faster when the parser recognizes a plus code.

Much faster search will be available in next Cruiser.

For all special searches: coordinates, plus codes, what3words, etc.

**Comment by DolomitesTom** [comment]
mostly  7 characters + location ... as search provider I have use OpenRouteService 

perfect, if you could sort this out that quick

**Comment by devemux86** [comment, marked as answer]
> Much faster search will be available in next Cruiser.

Implemented in [Cruiser 3.0.32](https://play.google.com/store/apps/details?id=gr.talent.cruiser).

**Reply by DolomitesTom** [reply]
yes, works much quicker.
